app/mixins/preload_data.js

```diff
@@ -1,48 +1,82 @@
-import Ember from 'ember';
-import AjaxPromise from '../utils/ajax-promise';
-import config from '../config/environment';
+import Ember from "ember";
+import AjaxPromise from "../utils/ajax-promise";
+import config from "../config/environment";
 
 export default Ember.Mixin.create({
-
-  preloadData: function(includePublicTypes) {
+  preloadData(includePublicTypes) {
     var promises = [];
     var isDonorApp = this.get("session.isDonorApp");
 
-    var retrieve = types => types.map(type => this.store.findAll(type, { reload: true }));
+    var retrieve = types =>
```

Comment:
@Darshan-Chauhan can we configure the `prettier` plugin ? I don't agree with all of its decisions 😆 
I admin it's a bit frustrating when a pull request has changes that I haven't made haha

Reply:
I agree that it's been frustrating 😄  .... I think if we'd used it from beginning than we wouldn't have this issue. 😅 
Should work fine once every file is done.

Reply:
@Darshan-Chauhan where is it configured ? So that I can match the configuration to my vscode prettier plugin : )

Reply:
@patrixr , it's not configured. You can add configuration if you want :) => https://prettier.io/docs/en/configuration.html